### PR TITLE
Add an option '-e' for user to run a shell script after each build

### PR DESCRIPTION
**Pull request by hanayashiki**
Usage:
```
cra-build-watch -e "echo 'build is done'"
```
Sometimes we may need to run routine after each build. I add this option to upload my build output to my remote server, and believe this would be useful for other uses.

### CODE REVIEW

**Comment by sandy2008** [comment]
Wow, it looks so useful.

File: scripts/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const fs = require('fs-extra');
 const path = require('path');
 const ora = require('ora');
 const assert = require('assert');
+const { exec } = require("child_process");
```
Comment (Nargonath) [inline scripts/index.js:10]:
```suggestion
const { exec } = require('child_process');
```

File: scripts/index.js [hunk 2]
```diff
@@ -154,6 +156,20 @@ fs.emptyDir(paths.appBuild)
...
+              return;
+            }
+            if (stderr) {
+              console.log(`stderr: ${stderr}`);
+              return;
+            }
+            console.log(`stdout: ${stdout}`);
+          });
+        }
+
```
Comment (Nargonath) [inline scripts/index.js:172]: I'd rather chain that as another then after this line: https://github.com/Nargonath/cra-build-watch/blob/acb74b48ffe39b25f68fa2ce73f6f93969946ee8/scripts/index.js#L161. This way your build path will also contain the `public` folder content.

Also as you're using the asynchronous version of `exec`, please wrap in a `Promise` to follow the current async chain.

**Comment by hanayashiki** [comment]
> Thanks for the addition 👍 . There needs to be some slight changes IMO but otherwise looks good. Also would you mind adding a mention of the new parameter to the README please?

Thank you for your suggestion. I've update in my latest commits. I also make use of https://www.npmjs.com/package/ora as you do to keep the log user-friendly and consistent. How does it look like now?

**Comment by hanayashiki** [comment]
Are we ok now to merge this to master?

**Comment by Nargonath** [comment]
@hanayashiki Please don't add another feature to your PR. I cannot squash the commits and release the feature atomically. I'll revert the commits about `runOnce`, please open a new PR if you want to add it.

**Comment by Nargonath** [comment]
Well it seems I can't push modifications to your PR, so please do it yourself otherwise I'll have to close the PR. Thanks.

**Comment by Nargonath** [comment]
I'm closing this PR in favor of https://github.com/Nargonath/cra-build-watch/pull/218. This other PR adds 2 hooks for after initial build and after each rebuild. You'll get the feature you were looking for and even more. Feel free to open another PR for your `runOnce` option so we can discuss it further.

Thanks for your work.

**Pull request closed without merging**